code/script.js

```diff
@@ -29,19 +37,109 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+// This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
-}
+};
 
-// Starts here
-const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
+// INITAL BOT GREETING! 
+const greetUser = () => {  
+  showMessage("Hello there, What's your name?", 'bot');  
+};
 
 // Set up your eventlisteners here
+form.addEventListener('submit', (event)=> {
+  event.preventDefault() // Prevents form to autorefresh
 
+const nameInput = document.getElementById('name-input').value;
+showMessage(nameInput, 'user');
+
+// Bot greets nameInput and ask for what drink
+chat.innerHTML += `<section class="bot-msg">
+    <img src="assets/bot.png" alt="Bot" />
+    <div class="bubble bot-bubble">
+        <p>Nice to meet you ${nameInput}! 👋 <br><br>
+        What type of drink would you like to order?</p>
+    </div>
+</section>`
+
+inputWrapper.innerHTML = `
+<button id="colaBtn" type="submit">Cola</button>
+<button id="fantaBtn" type="submit">Fanta</button>
+<button id="spriteBtn" type="submit">Sprite</button>
+`;
+
+let colaBtn = document.getElementById('colaBtn');
+colaBtn.addEventListener('click', () => cola())
+
+let fantaBtn = document.getElementById('fantaBtn');
+fantaBtn.addEventListener('click', () => fanta())
+
+let spriteBtn = document.getElementById('spriteBtn');
+spriteBtn.addEventListener('click', () => sprite())
+
+});
+
+// User choose Cola
+const cola = () => {
+  userReply(`I would love to order a Cola`);
+  setTimeout(() => botReply(`Great choice, Cola is amazing! Please select what kind of Cola below...`), 1000)
+
+       inputWrapper.innerHTML =
+        `<select id="select">
+          <option value selected disabled>Select what kind of Cola 👇</option>
+          <option value="regular">Regular</option>
+          <option value="zero">Zero</option>
+          <option value="vanilla">Vanilla</option>
+        </select>
+        <button class="selectBtn" id="colaKindBtn" type="submit">OK</button>
+        `
+        const colaKindBtn = document.getElementById('colaKindBtn');
+        colaKindBtn.addEventListener('click', () => {
+          botReply(`Thank you for your Cola-order! Close this window and your drink will be magical teleported to you 🧙‍♂️`)
+        }
+      )};
+
```

Comment:
If you feel like you want to and have time to adjust the ending of the bot, you can repeat the flavor selection from the user (userReply), and also add it to the end message.